src/client/Client.cc

```diff
@@ -6384,7 +6385,7 @@ void Client::tick()
     }
   }
 
-  if (mdsmap->get_epoch()) {
+  if (!mount_aborted && mdsmap->get_epoch()) {
```

Comment:
Yeah, we need it. Or we will hit the same inode ref leakage again.

If mount aborted, in the `tick()` we will skip all the code which need to use the session connections to send requests. Then the `tick()` will only do the local work like putting the inode refs, etc.

This could make sure that before the waiting for the `inode_map` to become empty, the timer is not cancelled.

Reply:
Sorry I forgot to mention, this PR will be related to defer putting the inode ref one https://github.com/ceph/ceph/pull/36580.